src/common/services/util-service.ts

```diff
@@ -23,7 +23,13 @@ export const getErrors = (values: any) => {
 };
 
 export const isFormFieldValid = (name: any, formik: any) => {
-  if (name.toLowerCase().includes("answer".toLocaleLowerCase())) {
+  if (name.toLowerCase().includes("answerInput".toLowerCase())) {
+    if (!formik.touched[name]) {
+      return !!(formik.errors[name] && formik.touched[name]);
+    } else {
+      return !!formik.errors[name];
+    }
+  } else if (name.toLowerCase().includes("answerRadio".toLowerCase())) {
```

Comment:
I would rethink the logic here. For instance, the first return is a bit confusing as the value will always be `!!(formik.errors[name] && false)` which is equivalent with `!!(false) => false`. This means that when the field is not touched the return value will be always false, hence meaning the form field is not valid.